Update old FLoC blog post (#8299)
GoogleChrome/web.dev: src/site/content/en/blog/floc/index.md

cookies or other tracking mechanisms. See [Digging into the Privacy Sandbox](/digging-into-the-privacy-sandbox)
for an overview of all the proposals.

> **This proposal needs your feedback.** If you have comments, [create an

Author: ```suggestion
If you have comments on this proposal, [create an
```
